apps/android/app/src/main/java/chat/simplex/app/model/ChatModel.kt

```diff
@@ -498,8 +500,14 @@ class CIMeta (
   }
 }
 
-// TODO
-fun getTimestampText(d: Instant): String = ""
+// TODO use old api?
+// TODO date for older timestamps
+@SuppressLint("NewApi")
+fun getTimestampText(d: Instant): String {
+  val dt = d.toLocalDateTime(TimeZone.currentSystemDefault()).toJavaLocalDateTime()
+  val formatter = DateTimeFormatter.ofPattern("HH:mm")
+  return dt.format(formatter)
+}
```

Comment:
we need to unify it with the time/date logic in the chat previews that @IanRDavies did - I will do that later to match the logic in iOS app.

In iOS app it shows the time if it is today OR if it is (yesterday AND the time is after 6pm AND the current time is before 12pm) - so, effectively, in the morning you know if it was the evening.

We can revise it to something else.